appveyor.yml

```diff
@@ -30,9 +30,9 @@ dotnet_csproj:
   file: '**\*.csproj;**\*.props;**\*.fsproj;**\*.xml'
   version: '{version}'
   package_version: '{version}'
-  assembly_version: '{version}'
-  file_version: '{version}'
-  informational_version: '{version}'
+  assembly_version: '{version}.0'

```

Comment:
shouldn't we change the version in line 8 accordingly and from here keep only reference?

Reply:
Thanks for fixing it